snowflake/assets/configuration/spec.yaml

```diff
@@ -141,6 +141,14 @@ files:
             - snowflake.billing
             - snowflake.storage
             - snowflake.logins
+      - name: use_deprecated_aggregation
```

Comment:
what do you think about making the option more descriptive, something like `aggregate_24_hours`? There could be instances where someone would prefer aggregating on the current day

Reply:
I was hesitant to use something like `aggregate_24_hours` ; as `aggregate_24_hours: false` does not give that much info on how the queries are aggregated. I'll change the name of the option though

Reply:
would aggregate_last_24_hours make make more sense?